### DISCUSSION TITLE
Export email evidence file

### DISCUSSION THREAD

**Discussion by thmme** [sepinf-inc/IPED#241] [Q&A] [answered]
Dear all, 

I would like to check the exportation for original emails files. In this case, is it possible to export the original file as .msg or .eml including all attachments?

Regards,

**Comment by lfcnassif** [comment, marked as answer]
If you are analyzing PST/OST files, they do not keep the original email, it is broken into several fields internally and it is not possible to recover the original eml as it was received. Rebuilding and exporting a "similar" eml file is possible, but that is not supported, and could confuse users, they could think it is the original mail. The html preview file generated by IPED makes it clear that it is not the original email.

Mbox and DBX mailboxes keep the original emails, so IPED extracts them and you can export if desired.
